app/views/secondary_content_links/index.html.erb

```diff
@@ -52,11 +52,7 @@
               label: {
                 text: "Base Path"
               },
-              hint: capture do %>
-                For example:
-                <a href="https://www.gov.uk/pay-vat" target="_blank" class="govuk-link">https://www.gov.uk/pay-vat</a>
-                or <a href="https://www.gov.uk/bank-holidays" class="govuk-link" target="_blank">/bank-holidays</a>
-              <% end,
+              hint: t("secondary_content_links.index.example"),
```

Comment:
Could we add the labels for each input to the yaml file too and I suggest we structure them based on their name instead of what their content so that, for example, if we decide to swap the hint for this field to say something else instead of showing an example we won't need to rename the yaml declaration.

```
secondary_content_links:
    index:
        base_path:
            label: Base Path
            hint: For example: https://www.gov.uk/pay-vat or /bank-holidays
```